src/obfl-specification.html

```diff
@@ -343,13 +343,56 @@ <h3 id="L5025">Auxiliary Flows</h3>
 of these containers, starting from 1 each time.</p>
 
 <h3 id="L5023">Simplex</h3>
-<p class="markup">In a sequence where duplex is not enabled, only the front side of sheets are counted and paginated. 
-The back side of sheets are neither counted nor paginated.</p>
+<p class="markup">In a sequence where duplex is not enabled, only the front side
+  of sheets are counted and paginated. The back side of sheets are neither
+  counted nor paginated.</p>
 
 <h3 id="L5024">Duplex</h3>
-<p class="markup">In a sequence where duplex is enabled, both sides of sheets are counted and paginated.
-However, if the last sheet of a duplex sequence only has content on the front side, 
-the back side is counted, but not <em>paginated</em>.</p>
+<p class="markup">In a sequence where duplex is enabled, both sides of sheets
+  are counted and paginated. However, if the last sheet of a duplex sequence
+  only has content on the front side, it depends on the values of the optional
+  attributes break-before and break-after on sequence, dynamic-sequence and
+  toc-sequence whether there is a back side, and if so, whether it is counted or
+  paginated. The default behavior is that there is a back side and it is counted
+  but not <em>paginated</em>.</p>
+
+<ins datetime="20200612T15:00:00">The last page of a sequence where duplex is
+enabled, may not be paginated and may not be counted.</ins>
+<p></p>
```

Comment:
It is only now that I noticed the "Backwards Incompatible Changes" section. Are all of these changes actually backwards incompatible? For example, the "external-reference" addition. Also the changes in this PR are not backwards incompatible as far as I know. At least in my proposal, I deliberately kept the default behavior the same.

Reply:
You're right. It is backwards compatible. You don't have to use the `break-after` element and the `page` value which makes it perfectly backward compatible. I will remove this `<ins ... />`.